templates/patterns/inpage_navigation/inpage_navigation.ui_patterns.yml

```diff
@@ -0,0 +1,27 @@
+inpage_navigation:
+  label: 'Inpage navigation'
+  description: 'Inpage navigation adds a navigation element and content element so when scrolling the active element is indicated in the nav part.'
+  fields:
+    title:
+      type: 'text'
```

Comment:
In 799 I'm leaving types without quotes because of reading on IDE, the type has a different color and improve my reading. This is a convention from myself and can be discussed if you want.

Reply:
Discussed with Francesco, he is agree doing in this way.